app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -64,14 +63,13 @@ class BrowserFragment : Fragment(), BackHandler {
     ): View? {
         require(arguments != null)
         sessionId = BrowserFragmentArgs.fromBundle(arguments!!).sessionId
-        isPrivate = BrowserFragmentArgs.fromBundle(arguments!!).isPrivateTab
 
         val view = inflater.inflate(R.layout.fragment_browser, container, false)
 
         toolbarComponent = ToolbarComponent(
             view.browserLayout,
             ActionBusFactory.get(this), sessionId,
-            isPrivate,
+            (activity as HomeActivity).browsingModeManager.isPrivate,
```

Comment:
Will this work with CustomTabs?

Reply:
It doesn't seem to break anything that isn't already broken on master. That being said, it seems like custom tabs are a bit buggy on master, so I'm filing a follow-up issue for that at #642.

Reply:
To clarify: custom tabs don't seem to be "actually" integrated yet.

Reply:
@sblatz They were :( Not sure what happened. This should still work for custom tabs.